### DISCUSSION TITLE
Invoice Noscript UI Extension Point

### DISCUSSION THREAD

**Discussion by rafael-xmr** [btcpayserver/btcpayserver#4531] [Plugins & integrations]
The ability to inject an `<iframe />` or a simple link url into the `invoice-noscript` page so that a plugin for coin swaps for example can still be enabled for customers even when they are browsing without javascript. trocador.app is built by scratch without javascript for example so it would work fine for those users.

**Comment by dennisreimann** [comment]
Thanks, I've added a PR for this: #4532
